README.md

```diff
@@ -108,26 +108,173 @@ Enables `QtWebEngine` functionality. For more details see the [example](./exampl
 
 This feature is disabled by default.
 
-## What if a binding for the Qt C++ API you want to use is missing?
+## What if a wrapper for the Qt C++ API is missing?
 
-It is quite likely that you would like to call a particular Qt function which is not wrapped by
-this crate.
+It is quite likely that you would like to call a particular Qt function which
+is not wrapped by this crate.
 
-In this case, it is always possible to access C++ directly from your rust code using the `cpp!` macro.
+In this case, it is always possible to access C++ directly from your rust code
+using the `cpp!` macro.
 
-Example: from [`examples/graph/src/main.rs`](./examples/graph/src/main.rs#L37), the struct `Graph` is a `QObject` deriving from `QQuickItem`,
-`QQuickItem::setFlag` is currently not exposed in the API but we wish to call it anyway.
+We strive to increase coverage of wrapped API, so whenever there is something
+you need but currently missing, you are welcome to open a feature request on
+GitHub issues or send a Pull Request right away.
+
+### Tutorial: Adding Rust wrappers for Qt C++ API
+
+This section teaches how to make your own crate with new Qt wrappers, and walk
+through a Graph example provided with this repository.
+
+First things first, set up your _Cargo.toml_ and _build.rs_:
+
+1. Add `qttypes` to dependencies. If you need latest unreleased changes, use
+    * `path = "../qttypes"` or
+    * `git = "https://github.com/woboq/qmetaobject-rs"`
+   otherwise, stick to recent versions published on [crates.io](versions).
+   ```toml
+   [dependencies]
+   qttypes = { version = "0.2", features = ["qtquick", "..."] }
```

Comment:
because of `"..."`  this line cannot be copy pasted

Reply:
Unfortunately, TOML doesn't have inline comments. I'll rewrite it in english to emphasis possible need for other features.